paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSheetActivityTest.kt

```diff
@@ -130,14 +134,60 @@ internal class CustomerSheetActivityTest {
         }
     }
 
+    @Test
+    fun `When savedPaymentMethods is not empty, payment method is visible`() {
+        runActivityScenario(
+            viewState = createSelectPaymentMethodViewState(
+                savedPaymentMethods = listOf(
+                    PaymentMethodFixtures.CARD_PAYMENT_METHOD,
+                )
+            ),
+        ) { scenario ->
+            scenario.onActivity {
```

Comment:
Looks like we always do the same thing in this block (scenario.onActivity), maybe we should just do this in the `runActivityScenario` method?

Reply:
Will address in the next PR!